civis/ml/_model.py

```diff
@@ -83,6 +83,12 @@ def wrapper(self):
 
 def _stash_local_dataframe(df, client=None):
     """Store data in a temporary Civis File and return the file ID"""
+    # Standard dataframe indexes do not have a "levels" attribute,
+    # but multiindexes do. Checking for this attribute means we don't
+    # need to import pandas to do error handling here.
+    if getattr(getattr(df, "index", None), "levels", None) is not None:
+        raise TypeError("CivisML does not currently support "
```

Comment:
This feels nitpicky, but could you remove the "currently"? We don't have any plans to support multi-indexes, and I don't want to mislead people. You might also add a suggestion in the error message that users could call `.reset_index` to get around this.